translations/pt-br/README.md

```diff
@@ -0,0 +1,175 @@
+Artigos que podem ajudar pessoas a falar em público
+
+Esse repositório contém conteúdo para melhoria contínua e evolução para falar em público. O primeiro foco é apresentações em eventos técnicos, mas muitas dicas poderão ser usadas de diferentes formas para falar em público em eventos e audiências.
+
+* [Contribuindo](#contribuindo)
+* [Recursos](#recursos)
+* [História e Base de conhecimento](#historia)
+* [Licença](#licença)
+
+## Contribuindo
+
+Conhece algum artigo sobre falar em público que você gostaria de compartilhar conosco? 
+
+Por favor, dê uma olhada em [CONTRIBUTING.md](./CONTRIBUTING.md) para saber como contribuir :-)
+
+Todos os contribuidores e participantes deste projeto (incluindo o [canal no IRC](https://webchat.freenode.net/?channels=%23public_speaking)[en]) aceitam em seguir o [Código de Conduta](./CODEOFCONDUCT.md).
+
+Por favor [fale conosco](https://github.com/airtonzanon/Public_Speaking/blob/master/CONTRIBUTING.md#for-more-help) se você tiver qualquer pergunta sobre o projeto ou como contribuir.
+
+## Recursos
+
+Dê uma olhada abaixo ou clique aqui para ir direto na sessão:
+
+* [Geral](#geral)
+* [Procurando um evento](#procurando-um-evento)
+* [Propondo palestras](#propondo-uma-palestra)
+* [Rejeições](#lidando-com-rejeições)
+* [Escrevendo a palestra](#escrevendo-palestras)
+* [Apresentando a palestra](#apresentando)
+* [Outros](#outros)
+
+### Geral
```

Comment:
These links are going to change far too frequently for them to be duplicated here. We'd forget to update the non-English document, making those non-English things second class citizens. We can't have that. Instead, probably best to remove all of the resources from here and link to the appropriate section in the main README instead.